### ISSUE TITLE
[Rule Request] after_comma and before_comma

### ISSUE THREAD

**Issue by toshi0383** [realm/SwiftLint#3120]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### New rule request

When `comma` rule is enabled, it's impossible to format code like this, wanting to align second element of tuple for readability.

```swift
let a = ("ab", 1)
let c = ("c",  0)
             ^^ two whitespaces after comma
```

Or maybe adding some comments. https://github.com/realm/SwiftLint/issues/2362

On the other hand, we need linter for whitespace before comma.

```swift
let a = ("a" ,  0)
            ^ whitespace before comma
```

To resolve this situation, I propose to have two separate rules called:

- `after_comma`
- `before_comma`

Both should be disabled by default to avoid conflict with `comma` which is enabled by default.
In case they conflict by users preference (`.swiftlint.yml`), `comma` rule supersedes.

#### after_comma.minimum

Minimum number of whitespaces allowed after comma.

```
let a = ("ab", 1) // allowed
let a = ("ab",1) // NOT allowed
let c = ("c",    0) // allowed
```

**Comment by christiankm** [comment]
I would say it's better to make the comma-rule configurable in this way, to allow any number of spaces, or just a minimum value.
